ooniprobe/View/TestResults/Footer/UploadFooterViewController.m

```diff
@@ -52,13 +55,71 @@ -(void)showModalHelp{
 }
 
 -(void)uploadResult{
```

Comment:
I am confused by the fact that I do not understand who's calling this function. I have tried to search for `uploadResult` but I have not found other matches apart this one.

Reply:
its called from
-(void)showModalHelp{
 and 
-(IBAction)upload{